### ISSUE TITLE
Start and End must be well ordered

### ISSUE THREAD

**Issue by Project-Magenta** [Microsoft/PTVS#3902]
# Process
devenv.exe

# Main Instruction
An unexpected error occurred

# Content
```
Build: 15.6.18045.1

System.ArgumentException: Start and End must be well ordered
   at Microsoft.PythonTools.SourceSpan.ValidateLocations(SourceLocation start, SourceLocation end)
   at Microsoft.PythonTools.Intellisense.LocationTracker.Translate(SourceSpan span, Int32 fromVersion, Int32 toVersion)
   at Microsoft.PythonTools.Intellisense.VsProjectAnalyzer.ConvertNavigations(Navigation[] navigations, LocationTracker translator, Int32 fromVersion, ITextSnapshot snapshot)
   at Microsoft.PythonTools.Intellisense.VsProjectAnalyzer.ConvertNavigations(Navigation[] navigations, LocationTracker translator, Int32 fromVersion, ITextSnapshot snapshot)
   at Microsoft.PythonTools.Intellisense.VsProjectAnalyzer.<GetNavigationsAsync>d__169.MoveNext()
--- End of stack trace from previous location where exception was thrown ---
   at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)
   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   at Microsoft.VisualStudioTools.UIThread.<>c__DisplayClass18_0`1.<<InvokeTask>b__0>d.MoveNext()
--- End of stack trace from previous location where exception was thrown ---
   at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)
   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   at Microsoft.PythonTools.Navigation.DropDownBarClient.<RefreshNavigationsFromAnalysisEntry>d__36.MoveNext()
--- End of stack trace from previous location where exception was thrown ---
   at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)
   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   at Microsoft.PythonTools.Navigation.DropDownBarClient.<Microsoft-PythonTools-Editor-IPythonTextBufferInfoEventSink-PythonTextBufferEventAsync>d__35.MoveNext()
--- End of stack trace from previous location where exception was thrown ---
   at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)
   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   at Microsoft.PythonTools.Infrastructure.VSTaskExtensions.<HandleAllExceptions>d__5.MoveNext()
```

**Comment by huguesv** [comment]
Thanks. This is a duplicate of https://github.com/Microsoft/PTVS/issues/3835. As mentioned there, try disabling the navigation bar (tools/options/text editor/python) to get around the issue.

@zooba I think we should fix in 15.6.

**Comment by huguesv** [comment]
We won't be fixing this in 15.6.
